### DISCUSSION TITLE
Autocomplete activating in annoying places

### DISCUSSION THREAD

**Discussion by justinluque** [LazyVim/LazyVim#5265] [Q&A] [answered]
Does anyone know any idea how to deal with this? Or make it less annoying?

I'm programming in C++ and whenever I finish a line, frequently I'm giving the option to autocomplete a random thing for what appears to be no reason. 

I think the plugin responsible is blink but I don't really know what I could possibly do to mitigate this. Any help is appreciated! 

![image](https://github.com/user-attachments/assets/665e1d0f-cc8b-4882-aa20-f5a8d8ffdecc)

**Comment by dpetka2001** [comment, marked as answer]
Try setting
```lua
return {
  {
    "saghen/blink.cmp",
    opts = {
      completion = {
        trigger = {
          show_on_accept_on_trigger_character = false,
          show_on_insert_on_trigger_character = false,
        }
      }
    }
  }
}
```
and see if that chages the behavior you observe.

**Comment by alvaroarenas** [comment]
At least for me, the annoyance was the english autocomplete.

<img width="400" alt="image" src="https://github.com/user-attachments/assets/d7d7de50-3bc7-4dcd-a629-5617c9231bf1" />


I solved it thx to [dpetka2001](https://github.com/dpetka2001) answer which pointed me to the correct plugin I had to configure.  I had to remove "buffer" from the default sources. Now my working configuration is below.

```lua
return {
  "saghen/blink.cmp",
  opts = {
    signature = { enabled = true },

    sources = {
      compat = {},
      default = { "lsp", "path", "snippets" },
      cmdline = {},
    },

    keymap = {
      preset = "enter",
      ["<C-y>"] = { "select_and_accept" },
    },
  },
}

```

**Comment by justinluque** [comment]
Thank you both for the responses!
